### ISSUE TITLE
Make ingore_log_data globally configurable

### ISSUE THREAD

**Issue by palkan** [palkan/logidze#109]
Let's make it possible to set `ignore_log_data` globally instead of per-model:

```ruby
# application.rb
Rails.application.config.logidze.select_ignore_log_data = true
```

**NOTE:** if we explicitly provide `ingore_log_data: false` with `has_logidze`, then it overrides the global setting.

Also, let's add an API to temporary change this global setting:

```ruby
# admin/application_controller.rb
class Admin::ApplicationController < AC::Base
  around_action :enable_logidze_log_data

  def enable_logidze_log_data
    Logidze.with_ignore_log_data(false) { yield }
  end
end
```

**Comment by DmitryTsepelev** [comment]
on it

**Comment by DmitryTsepelev** [comment]
Released as part of 0.10.0
